### DISCUSSION TITLE
Controlnet and other "built-in" extensions disabled on dev branch?

### DISCUSSION THREAD

**Discussion by MysticDaedra** [lllyasviel/stable-diffusion-webui-forge#326] [Q&A] [answered]
I switched to dev to take a look at the updates, and I noticed that the only extension dropdown I have now is one I've added, adetailer. Is this a bug (in which case I'll post a bug report), or is this as expected currently?

**Comment by freecoderwaifu** [comment, marked as answer]
Main seems more up to date than dev, so I'd stick with main at least for now.

**Reply by MysticDaedra** [reply]
Wait... main is more up to date than dev? Isn't that backwards lol?
